recipes/typer/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "typer" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "a8c49418f20c1d1b04a34424dded86cebf1f87042a8e448c6faa63cf38b9c0af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit-core
+  run:
+    - python
+    - click
+    - colorama
```

Comment:
It looks like `colorama` and `shellingham` are optional dependencies, but I guess that's ok to make this the equivalent of `pip install typer[all]`